staging/src/k8s.io/sample-apiserver/artifacts/example/rbac-bind.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: aggregated-apiserver-clusterrolebinding
```

Comment:
The binding must be sample-apiserver specific. Only the role should be generic.

Reply:
Fixed.